Gruntfile.js

```diff
@@ -94,7 +77,7 @@ module.exports = (grunt) => {
       },
     },
     eslint: {
-      target: SOURCES.concat('./tests'),
+      target: SOURCES.concat('./tests/**/*.?s'),
```

Comment:
too generic regex

Reply:
Please propose a regex

Reply:
Please note that the original one was leaving the content of the tests directory unchecked.

Reply:
I don't think this is a regex, this is some file globbing syntax. How about: `./tests/**/*.(js|ts)`

Reply:
If that doesn't work, then this existing one is fine -- but good to try and find one that is more tightly scoped.

Reply:
You are right it is not a regex, it is a list. Both .js and .ts added to the list